### DISCUSSION TITLE
button filled to represent battery charge state

### DISCUSSION THREAD

**Discussion by aaamoeder** [Clooos/Bubble-Card#971] [Feature requests]
covers and media player volume and more are represented by a partially filled bar.
Could this also be implemented for battery entities ? with extra options to have a gradient color fill according to the battery charge ?
This would enable me to finish my Car dashboard 👍

**Comment by SylvainDx** [comment]
MrBearPresident just share this : https://github.com/Clooos/Bubble-Card/discussions/928
if it can help ;)

**Reply by aaamoeder** [reply]
Thank you !!!
